numpyro/infer/hmc.py

```diff
@@ -374,12 +382,38 @@ class HMC(MCMCKernel):
     :param float step_size: Determines the size of a single step taken by the
         verlet integrator while computing the trajectory using Hamiltonian
         dynamics. If not specified, it will be set to 1.
+    :param inverse_mass_matrix: Initial value for inverse mass matrix.
+        This may be adapted during warmup if adapt_mass_matrix = True.
+        If no value is specified, then it is initialized to the identity matrix.
+        For a potential_fn with general JAX pytree parameters, the order of entries
+        of the mass matrix is the order of the flatten version of pytree parameters,
```

Comment:
nit, is this better?  

... of the flattened version of pytree parameters obtained with `tree_flatten`,